add section clients and styles for it

dizlord opened the pull request:
also add new icons to the sprite

dizlord committed:
add section clients and styles for it

also add new icons to the sprite

AntonyUser reviewed (commented):
Предлагаю h2 сделать скрытым, а то, что сейчас в h2 перенести в p. Так же, если вешаем на секцию класс клиенты, класс контейнера и листа нужно сделать согласно ВЕМ - cleints__container, clients__list
Либо на секцию и контейнер не задавать класс clients, а дать его на ul для возможного дальнейшего переиспользования 
По css, padding для контейнера предлагаю сделать 55рх 82 рх - чтоб скорректировать работу дизайнера

dizlord committed:
feature/clients

Remove section clients.
Change the title of section.
Add utils visually-hidden.
Change paddings for section

AntonyUser reviewed (commented):
Удачи у себя index.css и index.css.map, которые подтягивает sass
Идет конфликт файлов и не дает замержить

dizlord committed:
remove index.css and index.css.map
